mozperftest_tools/mozperftest_tools/side_by_side.py

```diff
@@ -593,6 +596,13 @@ def run(self, test_name="", new_test_name="", platform="", new_platform="", vism
                 "WARNING: Downloads will not be skipped as you are using something other "
                 "than the similarity metric (only supported for this metric)."
             )
+        if original:
+            artifacts_to_get = ["browsertime-videos-original"]
+        else:
+            artifacts_to_get = [
+                "browsertime-videos-annotated",
```

Comment:
Why do you need to download both of these when `--original` isn't used?

Reply:
if `original=True` then only `original` artifact is used, otherwise both `original` and `annotated`
`original=False` by default.

Reply:
original: If set, use the original videos in the side-by-side instead of the postprocessed videos

Reply:
Ok, but why do we need to download both the annotated, and the original videos if we're only going to be processing the annotated ones?

Reply:
if `original` is `True` then use only `original`, else use both.

Reply:
Currently the `original` flag causes the code to use only the original videos, and doesn't touch the annotated ones when it's provided. With the changes here, you're downloading both the annotated, and original videos, but you're only processing the annotated ones.

Reply:
sparky, can I have the line that is processing only the annotated ones despite downloading both, please?

Reply:
https://github.com/mozilla/mozperftest-tools/blob/master/mozperftest_tools/mozperftest_tools/side_by_side.py#L697-L698

Reply:
Previously there was only one artifact to download regardless of the original flag, but now there's two artifacts and the one we choose to download depends on this flag.

Reply:
Got it, thanks!